packages/firebase/index.d.ts

```diff
@@ -5473,6 +5473,57 @@ declare namespace firebase.storage {
      *     including if the object did not exist.
      */
     updateMetadata(metadata: firebase.storage.SettableMetadata): Promise<any>;
+    /**
+     * List items and sub-directories within this directory.
+     *
+     * "/" is treated as a path delimiter. Firebase storage does not support
+     * invalid object path that ends with "/" or contains two consecutive "//".
+     * All invalid objects in GCS will be filtered.
+     *
+     * @param options.maxResults If set, limits the total number of `prefixes`
```

Comment:
Please validate that this number doesn't exceed 1000. It would also be good to document this.

Reply:
Done